lib/y2_installbase.pm

```diff
@@ -182,10 +182,14 @@ highlight cursor one item down.
 
 sub move_down {
     # Sending 'down' twice, followed by 'up', scrolls to the intended item.
-    wait_screen_change { send_key 'down' } if is_sle('>=15-sp3');
     my $ret = wait_screen_change { send_key 'down' };
     last if (!$ret);    # down didn't change the screen, so exit here
-    wait_screen_change { send_key 'up' } if is_sle('>=15-sp3');
+    if (is_sle('>=15-sp3')) {
+        my $ret2 = wait_screen_change { send_key 'down' };
+        # If screen did not change with the second down we are on the last item.
+        # In that case do not press up so that it remenains selected.
+        wait_screen_change { send_key 'up' } if !$ret2;
```

Comment:
`if $ret2` ?

Reply:
I don't think so, if the screen changed, we send the key up, otherwise the screen did not change, so we are in the last item.

Reply:
ah actually yes, this is what I meat to write but did not in the code